jwk/ecdsa.go

```diff
@@ -157,25 +158,86 @@ func (k ecdsaPublicKey) Thumbprint(hash crypto.Hash) ([]byte, error) {
 	if err := k.Raw(&key); err != nil {
 		return nil, errors.Wrap(err, `failed to materialize ecdsa.PublicKey for thumbprint generation`)
 	}
+
+	xbuf := crvPointToFixedBuffer(key.X, key.Curve)
+	ybuf := crvPointToFixedBuffer(key.Y, key.Curve)
+	defer releaseCrvFixedBuffer(xbuf)
+	defer releaseCrvFixedBuffer(ybuf)
+
 	return ecdsaThumbprint(
 		hash,
 		key.Curve.Params().Name,
-		base64.EncodeToString(key.X.Bytes()),
-		base64.EncodeToString(key.Y.Bytes()),
+		base64.EncodeToString(xbuf),
+		base64.EncodeToString(ybuf),
 	), nil
 }
 
+var crvFixedBufferPool = sync.Pool{
+	New: func() interface{} {
+		return make([]byte, 66)
```

Comment:
It might be a good idea to use some readable const with explanation for this `66` value.

Reply:
Yeah, I was going meh, yeah, nah, etc about it. I guess I might as well.